packages/cli/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@dhis2/cli-utils-cypress",
+    "description": "DHIS2 Cypress configuration and utilities",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/dhis2/cli-utils-cypress.git",
+        "folder": "packages/cli"
+    },
+    "bin": {
+        "d2-utils-cypress": "./bin/d2-utils-cypress"
+    },
+    "main": "src/index.js",
+    "version": "2.2.1",
+    "author": "Viktor Varland <viktor@dhis2.org>",
+    "license": "BSD-3-Clause",
+    "private": false,
+    "engines": {
+        "node": ">=10"
+    },
+    "dependencies": {
+        "@dhis2/cli-helpers-engine": "^1.5.0",
+        "concurrently": "^5.2.0",
+        "cross-spawn": "^7.0.1",
+        "cypress": "^4.0.2",
+        "cypress-cucumber-preprocessor": "^2.0.1",
+        "fs-extra": "^8.1.0",
+        "inquirer": "^7.0.4",
+        "wait-on": "^4.0.2"
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "devDependencies": {
```

Comment:
`devDeps` are not necessary for packages, it's preferable to only have them in the root `package.json`.